lib/trento/application/integration/checks/adapter/mock_runner.ex

```diff
@@ -68,21 +74,58 @@ defmodule Trento.Integration.Checks.MockRunner do
     GenServer.cast(__MODULE__, {:set_expected_results, expected_results})
   end
 
-  defp send_execution_started_event(execution_id) do
-    # TODO: post event to callback
+  defp send_execution_started_event(execution_id, cluster_id) do
+    HTTPoison.post!(
+      "http://localhost:4000/api/runner/callback",
```

Comment:
Let's put a `TODO` maybe here. This hardcoded address looks something we need to change

Reply:
well, it's a mock we can make it dynamic ofc